OrbitCore/SymbolsManager.h

```diff
@@ -0,0 +1,50 @@
+#ifndef ORBIT_CORE_SYMBOLS_MANAGER_H_
+#define ORBIT_CORE_SYMBOLS_MANAGER_H_
+
+#include <atomic>
+#include <memory>
+
+#include "Message.h"
+#include "OrbitModule.h"
+#include "OrbitProcess.h"
+#include "OrbitSession.h"
+#include "TransactionManager.h"
+
+class CoreApp;
+
+namespace orbit {
+
+// The role of the SymbolsManager is to load debug symbol information either
+// from the developer's machine, or remotely on the target instance if symbols
+// weren't found locally.
+
+class SymbolsManager {
+ public:
+  SymbolsManager(CoreApp* core_app);
+
+  void LoadSymbols(std::shared_ptr<Session> session,
+                   std::shared_ptr<Process> process);
+  void LoadSymbols(const std::vector<std::shared_ptr<Module>>& modules,
+                   std::shared_ptr<Process> process);
+
+ private:
```

Comment:
deleted ctor-s should be in public: section.

Reply:
Done.